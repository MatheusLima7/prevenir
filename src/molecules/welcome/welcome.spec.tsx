import React from "react";
import { render } from "@testing-library/react-native";

import Welcome from "./welcome";

describe("Welcome Component", () => {
  it("should render component correctly", () => {
    const container = render(
      <Welcome title={"title"} text={"text"} org="org" />
    );

    expect(container).toMatchSnapshot();
  });
});
