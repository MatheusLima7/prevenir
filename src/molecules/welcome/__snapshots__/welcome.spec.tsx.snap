// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Welcome Component should render component correctly 1`] = `
<View
  style={
    {
      "textAlign": "center",
    }
  }
>
  <Text
    center={true}
    color="gray"
    size="large"
    style={
      [
        {
          "color": "#717171",
          "display": "flex",
          "fontFamily": "Quicksand-SemiBold",
          "fontSize": 24,
          "fontWeight": "600",
          "lineHeight": 30,
          "textAlign": "center",
        },
        {},
      ]
    }
    testId="text-component"
    type="title"
  >
    title
  </Text>
  <View
    style={
      {
        "marginTop": 20,
      }
    }
  >
    <Text
      center={true}
      color="gray"
      size="small"
      style={
        [
          {
            "color": "#717171",
            "display": "flex",
            "fontFamily": "Quicksand-Regular",
            "fontSize": 14,
            "fontWeight": "400",
            "lineHeight": 20,
            "textAlign": "center",
          },
          {},
        ]
      }
      testId="text-component"
      type="text"
    >
      text
       
      <Text
        color="gray"
        size="small"
        style={
          [
            {
              "color": "#717171",
              "display": "flex",
              "fontFamily": "Quicksand-SemiBold",
              "fontSize": 14,
              "fontWeight": "600",
              "lineHeight": 20,
            },
            {},
          ]
        }
        testId="text-component"
        type="title"
      >
        org
      </Text>
    </Text>
  </View>
</View>
`;
